app/models/post.rb

```diff
@@ -2,9 +2,10 @@ class Post < ApplicationRecord
   validates :title, presence: true, length: { maximum: 250 }
   validates :comments_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
   validates :likes_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
-  belongs_to :user, class_name: 'User', foreign_key: 'author_id'
-  has_many :comments
-  has_many :likes
+
+  belongs_to :author, class_name: 'User', foreign_key: :author_id
+  has_many :comments, foreign_key: :post_id
+  has_many :likes, foreign_key: :post_id
```

Comment:
- [ ] I'm still getting this issue which the previous reviewer asked for a fix. Please make sure you run locally all the pages successfully before asking for a review.

Reply:
Hi @Deepakdanger,

I have made the changes, please check